cfme/tests/test_replication.py

```diff
@@ -29,9 +54,9 @@ def test_replication_powertoggle():
     pass
 
 
-@pytest.mark.manual
 @pytest.mark.tier(2)
-def test_replication_appliance_add_single_subscription():
+def test_replication_appliance_add_single_subscription(configured_appliance,
```

Comment:
@mnadeem92, such use case is already covered in Multi-Region FA by many test cases. we need to review replication test cases.